### PR TITLE
Challenge/hours to minutes

### LINKED ISSUE #21: Convert Hours to Minutes
## A Function that converts Hours to Minutes ##

---
It takes in a measure of time in Hours  as input , converts it to its equivalent in minutes and prints gives the corresponding value in minutes.

>For example:
Input in Hours = 2
Output in Minutes = 120

### PR DESCRIPTION
**Pull request by tosan-okome**
Wrote a function to convert a given value in hours to minutes. 

Resolves #21 

Here's a review checklist: 

### Files

- [x] The file name describes the function's behavior
- [x] There is a module docstring in the function file
- [x] The test file's name matches the function file name -
  `/tests/test_file_name.py`
- [x] There is a module docstring in the tests file

### Unit Tests

- [x] The test class has a helpful name in PascalCase
- [x] The test class has a docstring
- Every unit test has
  - [x] A helpful name
  - [x] A clear docstring
  - [x] Only one assertion
  - [x] There is no logic in the unit test
- [x] All tests pass
- [x] There are tests for defensive assertions
- [x] There are tests for boundary cases

### Function Docstring

- [x] The function's behavior is described
- The function's arguments are described:
  - [x] Type
  - [x] Purpose
  - [x] Other assumptions (eg. if it's a number, what's the expected range?)
- The return value is described
  - [x] Type
  - [x] Other assumptions are documented
- The defensive assertions are documented using `Raises:`
  - [x] Each assumption about an argument is checked with an assertion
  - [x] Each assertion checks for _only one_ assumption about the argument
- [x] Include 3 or more (passing!) doctests

### The Function

- [x] The function's name describes it's behavior
- [x] The function's name matches the file name
- [x] The function has correct type annotations
- [x] The function is not called in the function file

### Do's

- [x] Variable names help to understand the strategy
- [x] Any comments are clear and describe the strategy
- [x] Lines of code are spaced to help show different stages of the strategy

### Don'ts

- [x] The function's strategy _is not_ described in the documentation
- [x] Comments explain the _strategy_, **not** the _implementation_
- [x] The function _does not_ have more comments than code
  - If it does, consider finding a new strategy or a simpler implementation

## Implementation

- [x] The code passes the formatting checks
- [x] The code passes all Ruff linting checks
- [x] The code has no (reasonable) Pylint errors
  - In code review, you can decide when fixing a Pylint error is helpful and
    when it's too restricting.
- [x] Variables are named with snake_case
- [x] Variable names are clear and helpful
- [x] The code follows the strategy as simply as possible
- [x] The implementation is as simple as possible given the strategy
- [x] There are no commented lines of code
- [x] There are no `print` statements anywhere
- [x] The code includes defensive assertions
- [x] Defensive assertions include as little logic as possible

### CODE REVIEW

**Review by Safiya-hash — COMMENTED** [review]
The code was easy to understand, well documented and unittests were helpful. 
Nice job done, you have covered edge cases, assertion errors, standard unit tests and no problem with CI checks.
Congrats 🎊 
Thanks for clean job ✨

**Review by Safiya-hash — DISMISSED** [review]
The code was easy to understand, well documented and unittests were helpful.
Nice job done, you have covered edge cases, assertion errors, standard unit tests, followed the review checklist and no problem with CI checks.
Therefore, I am approving the pull request merge, congrats 🎊
Thanks for clean job ✨

**Review by Safiya-hash — DISMISSED** [review]
.

File: solutions/hours_to_minutes.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+        Raises:
+        TypeError:
+            If the input is not a number (int or float).
+
+        ValueError:
+            If the input is negative.
+
+        Examples:
+            >>> hours_to_minutes(4)
```
Comment (Safiya-hash) [inline solutions/hours_to_minutes.py:35]: As the outputs of the doctests are integers/floats, it might be good to use exact number instead of number inside the parathesis for a clearer view. What do you think? 💭

Comment (tosan-okome) [reply solutions/hours_to_minutes.py:35]: I don't think I understand. Can you elaborate?

Comment (Safiya-hash) [reply solutions/hours_to_minutes.py:35]: Yeah sure, I meant that the current doctests results are integers inside parenthesis, I suggest we shift to using exact numbers for these outputs for example:
>>> hours_to_minutes(4)
4
>>> hours_to_minutes(0.5)
            30
 >>> hours_to_minutes(24)
  1440
>>> hours_to_minutes(12)
720
Your work still is flowless if you don't do it, I just thought it would make it clearer.

Comment (tosan-okome) [reply solutions/hours_to_minutes.py:35]: Oh. That makes a lot of sense. Thanks for pointing it out.

Comment (Safiya-hash) [reply solutions/hours_to_minutes.py:35]: Thanks for being flexible @tosan-okome, I am going to resolve the conversation and merge it to the main best of luck.

Comment (tosan-okome) [reply solutions/hours_to_minutes.py:35]: Fixed!

**Review by Safiya-hash — DISMISSED** [review]

**Review by Safiya-hash — APPROVED** [review]

**Review by Safiya-hash — DISMISSED** [review]

**Review by Safiya-hash — DISMISSED** [review]
After encountering several errors, hopefully this time it will merge.

**Review by Safiya-hash — APPROVED** [review]
